delphi_esr_msgs/msg/EsrEthTx.msg

```diff
@@ -1,19 +1,19 @@
-Header header
+std_msgs/Header header
 
 # ESR Track Msg
-uint16	      xcp_format_version
-uint16        scan_index
-uint16        tcp_size
-uint8         xcp_scan_type
-uint16        look_index
-uint16        mmr_scan_index
-float32         target_report_host_speed
-float32         target_report_host_yaw_rate
-uint32        xcp_timestamp
-uint8         release_revision
-uint8         promote_revision
-uint8         field_revision
-uint8        target_report_count
+uint16            xcp_format_version
```

Comment:
Is this a requirement for ROS2 linting, or just a formatting cleanup?
I'm completely new to anything ROS2 :)

Reply:
Using spaces instead of tabs in `.msg` or `.srv` files is a requirement for ROS2. The converter that translates `.msg` and `.srv` files to `.idl` files chokes on tabs.

Reply:
Good to know, thanks!